MemberExpectations.md

```diff
@@ -0,0 +1,59 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the
+[Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
+but that supports the vision, mission and operating principles of the Node.js project.
+
+It is understood that members will have individual opinions and will express
+their individual opinions during discussions within the organization. All
+contributors are expected to be respectful of these different viewpoints,
+and to work collaboratively with one another in a manner that constructively
+elevates all contributors.
+
+When an individual contributor chooses to accept an invitation to participate
+in a leadershp role, they implicitly take on an additional obligation to protect
+and be respectful of both the project and all other contributors.  When
+decisions are made within the established guidelines and policies
+of the project, those in leadership roles have a responsibility to uphold
+and respect the decision even if they disagree with it. This is especially
+important in external communications, for example in social media. Should
+the member be unwilling or unable to do so, then they should step down and
+resign their leadership position. This does not mean that decisions cannot
+be revisited and discussed within the team at a later time.
+
+Members of our leadership groups must also conduct themselves in a
+professional and respectful manner. Some general guidelines include:
+
+- Remediate quickly when you realize you made a mistake. Leaders are human,
+  and they will make mistakes - however they should act swiftly to
+  acknowledge mistakes and correct them. Most often this can mitigate
+  any damage done.
+- Aim to remediate first and then discuss.  If other members of the
+  team express concerns about actions, acknowledge their concerns by
+  stopping the actions in question and then discuss within the team
+  to come to a common agreement.
+- Treat all community members with respect, consideration, and highest
+  standards of ethical conduct.  
+- Valuing a diversity of views and opinions. Strive to avoid preferential
```

Comment:
Nit: Keep verb tense consistent across these items. So `Value a diversity of views and opinions.` rather than `Valuing a diversity....`

Reply:
More than a nit: Remove `Strive to`.

Reply:
ok